extensions/cms/package.json

```diff
@@ -253,11 +253,12 @@
           "description": "%cms.connectionOptions.trustServerCertificate.description%",
           "groupName": "%cms.connectionOptions.groupName.security%",
           "valueType": "boolean",
-          "defaultValue": null,
+          "defaultValue": "false",
```

Comment:
Any reason not to leave this as the default?  Does the driver default to false if this is left blank or is it server config dependent?

Reply:
The driver defaults to false, but if this is not set, the connection dialog defaults it to true.

Reply:
I wonder if we should leave null (empty) as a possible value for these select boxes. Because none of them are "required" - so leaving it blank should be equivalent to "I just want whatever the driver default is". This would be important for showing connection info for entered connections - because in those cases we don't want to show options having values that may end up being different than what the actual connection string would result in if entered into a SqlConnection directly. 

I will say in this case though we should probably show false to make it very clear what the default we're setting is - I just don't know if we want to require that for all the other boolean options.

Reply:
For these 2 properties I'd say we need to populate defaults so it's visible why connection failed, since we're enabling encryption by default as a new change. For other advanced properties it may not be required to show pre-populated defaults and let driver take that call. Its very rare defaults would change in the driver, but if they do, it may not be possible to keep track in ADS for every property.